cfg/cfg.go

```diff
@@ -24,6 +22,54 @@ type Config struct {
 	Validation_level_legacy validate.LevelLegacy
 	Validation_level_m20    validate.LevelM20
 	Validate_order          bool
+	BlackList               []string
+	Aggregation             []*AggregationConfig
+	Route                   []*RouteConfig
+	Rewriter                []*RewriterConfig
+}
+
+type AggregationConfig struct {
+	Function string
+	Regex    string
+	Format   string
+	Interval int
+	Wait     int
+}
+
+type RouteConfig struct {
+	Key          string
+	Type         string
+	Prefix       string
+	Substr       string
+	Regex        string
+	Destinations []string
+
+	// grafanaNet & kafkaMdm
+	SchemasFile  string
+	OrgId        int
+	BufSize      int
+	FlushMaxNum  int
+	FlushMaxWait int
+	Timeout      int
+
+	// grafanaNet
+	Addr        string
+	ApiKey      string
+	Spool       bool
+	SslVerify   bool
+	Concurrency int
+
+	// kafkaMdm
+	Broker      string
+	Topic       string
+	Codec       string
+	PartitionBy string
+}
+
+type RewriterConfig struct {
+	Old string
+	New string
+	Max int
```

Comment:
for these types, I think the following advice applies:

> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.

(https://blog.golang.org/package-names)

so I think we can name these simply Rewriter, Route and Aggregation, since they are in the cfg package it is clear they deal with configuration.

but if you rather not, that's fine with me too. I don't feel strongly about it.